java/src/main/java/com/powsybl/dataframe/network/NetworkDataframes.java

```diff
@@ -51,7 +51,8 @@ public static NetworkDataframeMapper getDataframeMapper(DataframeElementType typ
     private static Map<DataframeElementType, NetworkDataframeMapper> createMappers() {
         Map<DataframeElementType, NetworkDataframeMapper> mappers = new EnumMap<>(DataframeElementType.class);
         mappers.put(DataframeElementType.SUB_NETWORK, subNetworks());
-        mappers.put(DataframeElementType.BUS, buses());
+        mappers.put(DataframeElementType.BUS, buses(false));
```

Comment:
cant we use the dataframeContext ? or it is to complicated for a simple thing ?

Reply:
The first version I coded was using the dataframe context, this is why in previous PR I also made it accessible when creating the element list. I was at the end simpler to just create a new dataframe type and more consistent with the way it is exposed on python side with a dedicated method.